web/app/js/components/Navigation.jsx

```diff
@@ -245,6 +298,14 @@ class NavigationBase extends React.Component {
               <ListItemIcon><LibraryBooksIcon /></ListItemIcon>
               <ListItemText primary="Documentation" />
             </ListItem>
+            { this.menuItem("/community", "Community",
+              <Badge
+                classes={{ badge: classes.badge }}
+                invisible={this.state.hideUpdateBadge}
+                badgeContent="1">
+                <SentimentVerySatisfiedIcon fontSize="small" />
```

Comment:
change this to `medium` to align the menu items?

![Screen Shot 2019-03-08 at 2 30 11 PM](https://user-images.githubusercontent.com/236915/54059647-b9de9d80-41ae-11e9-8bdd-89d586a6bc29.png)

Reply:
Good 👁, I just removed the attribute to restore the default size.